spec/fixtures/vcr_cassettes/Stripe-v10.11.0/Spree_Gateway_StripeSCA/_credit/refunds_the_payment.yml

```diff
@@ -303,20 +307,20 @@ http_interactions:
           "canceled_at": null,
           "cancellation_reason": null,
           "capture_method": "automatic",
-          "client_secret": "pi_3Ooefi4DenidF2LU1DyZ34Mm_secret_w379gPpBGChVwBEUepP0qlRB0",
+          "client_secret": "pi_3OqeRV3UJ5brpc2a0jgoatoG_secret_BSfmlMO7KoJyiterLxC48ygb5",
```

Comment:
The word "secret" raised a flag for me 😉 
Is this environmental (something you have stored in `.env.test.local`)? 

Perhaps it's coming from your test account, and it just shows that the test can run independently with either yours or Gaetan's account, which is great. If there's no risk posed by it being exposed, then we can probably just leave it.

Reply:
Please see [this comment](https://github.com/openfoodfoundation/openfoodnetwork/pull/12252/commits/5653d542f696badbd30829d44945b152ab5a9d70#r1516342069).